src/modules/InvolvementUI.js

```diff
@@ -0,0 +1,101 @@
+import InvolvementService from './InvolvementService.js';
+import { Comment, Reserve } from './Modals.js';
+
+const getDetails = document.querySelector('.commentList');
+const hiddenLaunchId = document.getElementById('launch-id');
+const commentsCount = document.querySelector('.comments-count');
+const txtname = document.getElementById('txt-visitor');
+const txtcomment = document.getElementById('txt-comment');
+const btnSaveComment = document.getElementById('btn-save-comment');
+
+const getDetails1 = document.querySelector('.reserveList');
+const hiddenLaunchId1 = document.getElementById('ilaunch-id');
+const reserveCount = document.querySelector('.reserve-count');
+const username = document.getElementById('username');
+const startDate = document.getElementById('start_date');
+const endDate = document.getElementById('end_date');
+const btnSaveReserve = document.getElementById('btn-save-reserve');
+btnSaveComment.addEventListener('click', async () => { await InvolvementUI.postComment(); });
+btnSaveReserve.addEventListener('click', async () => { await InvolvementUI.postReservation(); });
+
+export default class InvolvementUI {
+  static renderComments = async (launchId) => {
+    document.getElementById('reserve').style.display = 'none';
+    document.getElementById('comment').style.display = 'block';
+    hiddenLaunchId.value = launchId;
+    getDetails.innerHTML = ' ';
+    await InvolvementService.getComments(launchId)
+      .then((comments) => {
+        commentsCount.innerText = InvolvementService.getCommentsCount(comments);
```

Comment:
- [ ] I think it's better to create a specific function to calculate the comments counter. A counter function should look for specific DOM elements and make the counting based on what is actually displayed on the page. Don't forget these count functions need to be covered with unit tests using Jest. Don't forget the case when the count is zero.